boards/cc2650stk/board.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Leon George
+ * (who won't go after you should you copy any part of this file ;-)
```

Comment:
this you should take out, copyright is copyright, so no need to argue about it...